lms/services/digest.py

```diff
@@ -14,7 +16,22 @@
     User,
 )
 from lms.services.h_api import HAPI
-from lms.services.mailchimp import EmailRecipient, EmailSender, MailchimpService
+from lms.services.mailchimp import (
+    EmailRecipient,
+    EmailSender,
+    MailchimpError,
+    MailchimpService,
+)
+
+LOG = logging.getLogger(__name__)
+
+
+class SendDigestsError(Exception):
+    """An error when sending a batch of email digests."""
+
+    def __init__(self, errors):
+        super().__init__(errors)
+        self.errors = errors
```

Comment:
For the reason for both passing `errors` to `super().__init__()` and setting it as `self.errors` see https://www.seanh.cc/2019/06/20/python-custom-exception-classes/